internal/controller/streaming_manager.go

```diff
@@ -144,6 +156,16 @@ func (sm *streamManager) BootUpStreamAndReconnect(ctx context.Context, cancel co
 		return err
 	}
 	snapshotCompleted.Done()
+	if ciliumFlowManager.client != nil {
+		go func() {
+			err = ciliumFlowManager.listenToFlows(ctx, *sm)
```

Comment:
This method returns an error whenever anything wrong happens, e.g. the RPC to Hubble or the RPC to CS is closed. And nothing calls this ever again.
So in case any error happens, this stops collecting and sending flows, until the pod is restarted.
That's not OK. This must retry connecting forever.